vms/platformvm/txs/tx.go

```diff
@@ -74,14 +75,29 @@ func (tx *Tx) ID() ids.ID    { return tx.id }
 func (tx *Tx) UTXOs() []*avax.UTXO {
 	outs := tx.Unsigned.Outputs()
 	utxos := make([]*avax.UTXO, len(outs))
-	for i, out := range outs {
+	// getting lockState that was applied in this tx (if were any)
```

Comment:
No chance to overwrite in camino_ file? If not, license header must be adjusted.